File: demo/Commons/src/main/java/com/example/commons/entity/FileList.java
package com.example.commons.entity;

import com.example.commons.groups.Insert;
import com.example.commons.groups.Update;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;
import javax.validation.constraints.NotNull;

import java.io.Serializable;
import java.util.Date;

@Data
public class FileList implements Serializable {
    @NotNull(message = "id不能为空", groups = { Update.class})
    private int id;
    private String title;
    private String content;
    private String fileName;
    @NotNull(message = "图片链接不能为空", groups = {Insert.class, Update.class })
    private String fileUrl;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private Date fileDate;
    private int isAdmin;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private Date createdAt;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private Date updatedAt;
}

File: php-laravel/app/Models/FileList.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class FileList extends Model
{
    protected $table = 'file_list';

    protected $fillable = [
        'title',
        'content',
        'file_name',
        'file_url',
        'file_date',
        'is_admin'
    ];

    /**
     * 为日期属性设置统一的序列化格式
     */
    protected function serializeDate(\DateTimeInterface $date)
    {
        return $date->format('Y-m-d H:i:s');
    }
}
